pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -270,6 +333,14 @@ public String getControllerPort() {
     return (String) getProperty(CONTROLLER_PORT);
   }
 
+  public String getControllerHttpsPort() {
+    if (!containsKey(CONTROLLER_HTTPS_PORT)) {
+      return "443";
```

Comment:
Can you please add a static final for the default port like the other configurations?